python-test/features/steps/agentsProvider.py

```diff
@@ -24,14 +25,19 @@ def create_agent(context):
     context.agent_id = context.agent_data['id']
     context.agent_channel_id = context.agent_data['channel_id']
 
-@when('Build agent container')
-def build_agent(context):
-    iface = 'mock'
-    provision_agent_command = f"docker run -d --net=host -e ORB_CLOUD_ADDRESS=beta.orb.live -e ORB_CLOUD_MQTT_ID={context.agent_id} -e ORB_CLOUD_MQTT_CHANNEL_ID={context.agent_channel_id} -e ORB_CLOUD_MQTT_KEY={context.agent_key} -e PKTVISOR_PCAP_IFACE_DEFAULT={iface} ns1labs/orb-agent"
-    context.id_terminal = functions.send_terminal_commands(provision_agent_command, '[')
-    assert len(context.id_terminal) > 0 
-
-@then("Agente should be online")
+@when('Run agent container')
+def run_agent(context):
+    iface = os.getenv('IFACE', 'mock')
+    context.orb_address = os.getenv('ORB_ADDRESS', 'beta.orb.live')
+    context.version_image = os.getenv('AGENT_VERSION')
```

Comment:
let's make this `AGENT_DOCKER_TAG` instead of `AGENT_VERSION`, because they are not quite the same thing: the tag may be `develop` for example which could be any version.